src/main/java/de/dhbw/mh/rinne/AstBuilder.java

```diff
@@ -61,6 +62,15 @@ public AstNode visitVariableReference(RinneParser.VariableReferenceContext ctx)
     // Team 1
 
     // Team 2
+    @Override
+    public AstNode visitFunctionCall(RinneParser.FunctionCallContext ctx) {
```

Comment:
You're currently implementing the rule `expression: funcCall`, but what you actually need to implement is the `funcCall` rule itself:

```g4
funcCall
  : BITTE functionName=IDENTIFIER (MIT (actualParameters+=expression)+)?
  ;
```

To match this rule, you should override the corresponding visitor method:

```suggestion
    public AstNode visitFuncCall(RinneParser.FuncCallContext ctx) {
```